server/game/cards/06-CotE/SageOfGiseiToshi.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes } = require('../../Constants');
+
+class SageOfGiseiToshi extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Move home, then move character home',
+            condition: context => this.game.isDuringConflict() && context.player.opponent && context.player.opponent.honor < context.player.honor,
```

Comment:
This ability targets before resolving, so the target: should be here, not in the then

Reply:
Gotcha, I tried it like that at first and it gave me a bit of trouble, but think I've sussed it now.  I've pushed a new version.